asynq/async_task.pxd

```diff
@@ -88,15 +69,15 @@ cdef class AsyncTask(futures.FutureBase):
     cdef _enter_context(self, context)
     cdef _leave_context(self, context)
     @cython.locals(i=int)
-    cdef _pause_contexts(self)
+    cpdef _pause_contexts(self)
     @cython.locals(i=int)
-    cdef _resume_contexts(self)
+    cpdef _resume_contexts(self)
```

Comment:
the scheduler needs to call these methods but only knows at runtime whether the current task is an `AsyncTask` -- so these unfortunately have to be `cpdef`